src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -1881,13 +1892,38 @@ public void canPostSynchronousBatch() throws Exception {
     JsonObject uprooted         = instancesArray.getJsonObject(0);
     JsonObject smallAngryPlanet = instancesArray.getJsonObject(numberOfInstances / 2);
     JsonObject temeraire        = instancesArray.getJsonObject(numberOfInstances - 1);
+
     assertExists(uprooted);
     assertExists(smallAngryPlanet);
     assertExists(temeraire);
 
     assertNotSuppressedFromDiscovery(instancesArray);
   }
 
+  @Test
+  public void newSynchronousBatchInstancesShouldHaveInitialStatusDate() throws Exception {
+    JsonArray instancesArray = new JsonArray();
+    
+    for (int i = 0; i < 3; i++) {
+      instancesArray.add(smallAngryPlanet(UUID.randomUUID()));
+    }
+
+    JsonObject instanceCollection = new JsonObject().put(INSTANCES_KEY, instancesArray);
+
+    CompletableFuture<Response> createCompleted = new CompletableFuture<>();
+    client.post(instancesStorageSyncUrl(""), instanceCollection, TENANT_ID, ResponseHandler.empty(createCompleted));
+    assertThat(createCompleted.get(30, SECONDS), statusCodeIs(HttpStatus.HTTP_CREATED));
```

Comment:
Could it be worth extracting a method here as well
```suggestion
    final var createCompleted = createInstancesBatch(batchRequest);
    
    assertThat(createCompleted.get(30, SECONDS), statusCodeIs(HttpStatus.HTTP_CREATED));
```